2-mandatory/2-retrying-random-numbers.js

```diff
@@ -11,6 +11,14 @@ function generateRandomNumber() {
 
 function getRandomNumberGreaterThan50() {
     // TODO - implement using a do-while loop
+    let counter=0;
+    do{
+       counter=generateRandomNumber();
+    }
+    while(counter<50);
```

Comment:
nice do-while loop implementation :) remember to use `<=` here rather than just `<` as the returned number needs to be greater than 50

Reply:
Excuse me, the task says in the 2nd line that ' use ```< 50```'. Despite of this should we still use ```<=```?